docs/extensibility/custom-resources.md

```diff
@@ -109,6 +109,9 @@ Press <kbd>Ctrl</kbd>+<kbd>C</kbd> to shutdown the app (you can close the browse
     dotnet add ./MailDev.Hosting/MailDev.Hosting.csproj package Aspire.Hosting --version 8.0.0-preview.7.24251.11
     ```
 
+    > [!IMPORTANT]
+    > The version you specify here should match the version of the .NET Aspire workload installed.
```

Comment:
(which ideally, both that and the workload would be the same version, but that is not always the case)

Reply:
That's what is being added... it will always match itself 🤣. The .NET CLI command is adding a package reference to version `8.0.0`.

Reply:
Right after the project is created, then yes, they will match of course. But think about the scenario where I have one workload version, create the project and push it to a repo, and you have a different workload version and pull the repo, and want to work on a client library (hosting package). This is what I meant that you'll want to match the version of the packagereference (which will match my workload as opposed to yours) instead of the version of your workload.

Reply:
We'll need to change the versions back in a few days ;)

Reply:
Shhhhh